compiler/commands.nim

```diff
@@ -307,7 +307,9 @@ proc testCompileOption*(conf: ConfigRef; switch: string, info: TLineInfo): bool
   of "fieldchecks": result = contains(conf.options, optFieldCheck)
   of "rangechecks": result = contains(conf.options, optRangeCheck)
   of "boundchecks": result = contains(conf.options, optBoundsCheck)
-  of "refchecks": result = contains(conf.options, optRefCheck)
+  of "refchecks":
```

Comment:
also update comment in options.nim:
```nim
    optOwnedRefs              # active if the Nim compiler knows about 'owned'.
=>
    optOwnedRefs              # deprecated: active if the Nim compiler knows about 'owned'.
```
(but please double check whether `optOwnedRefs` should indeed be deprecated)

ditto for `optRefCheck`